src/views/index-js.mustache

```diff
@@ -2633,13 +2633,17 @@ function getPokemonPopupContent (pokemon) {
             }
         });
     }
+    let pokemonIdString = '' + pokemon.pokemon_id + '-' + pokemon.form_id;
```

Comment:
```suggestion
    let pokemonIdString = '' + pokemon.pokemon_id + '-' + pokemon.form;
```
Even after working on RDM for almost 2 years I always thought it should be `form_id`, but.... it's just `form` for pokemon lol